pkg/logicalbackup/logicalbackup.go

```diff
@@ -66,15 +66,62 @@ type LogicalBackuper interface {
 	Wait()
 }
 
+type BackupTables struct {
+	sync.RWMutex
+	data map[dbutils.Oid]tablebackup.TableBackuper
+}
+
+func NewBackupTables() *BackupTables {
+	return &BackupTables{sync.RWMutex{}, make(map[dbutils.Oid]tablebackup.TableBackuper)}
+}
+
+func (bt *BackupTables) GetByOid(oid dbutils.Oid) tablebackup.TableBackuper {
+	bt.RLock()
+	defer bt.RUnlock()
+
+	return bt.data[oid]
+}
+
+func (bt *BackupTables) GetByOidIfExists(oid dbutils.Oid) (result tablebackup.TableBackuper, ok bool) {
```

Comment:
Makes no sense to me. GetByOidIfExists returns value or boolean false if the value doesn't exist. GetByOid would return a nil interface, which we never check, since our assumption is that Oid must exist when we call it. I didn't call it GetByOidMustExist because it would be too verbose, and also since it is usually reserved for panic behavior when the condition that must be satisfied is not.